Resources/public/js/views/dashboard/ez-dashboardblockallcontentview.js

```diff
@@ -0,0 +1,149 @@
+            this.events = Y.merge(this.events, EVENTS);
+
+            this._set('identifier', BLOCK_IDENTIFIER);
+            this.get('container').addClass(this._generateViewClassName(Y.eZ.DashboardBlockBaseView.NAME));
+        },
+
+        /**
+         * Renders the dashboard block view
+         *
+         * @method render
+         * @return {eZ.DashboardBlockAllContentView} the view itself
+         */
+        render: function () {
+            this.get('container').setHTML(this.template()).addClass(CLASS_LOADING);
+
+            return this;
+        },
+
+        /**
+         * Shows the row options
+         *
+         * @method _uiShowOptions
+         * @protected
+         * @param event {Object} event facade
+         */
+        _uiShowRowOptions: function (event) {
+            this._uiHideRowOptions();
+
+            event.currentTarget.addClass(CLASS_ROW_SELECTED);
+
+            this._clickOutsideHandler = this.get('container').one(SELECTOR_CONTENT).on('clickoutside', Y.bind(this._uiHideRowOptions, this));
+        },
+
+        /**
+         * Clears the click outside handler
+         *
+         * @method _clearClickOutsideHandler
+         * @protected
+         */
+        _clearClickOutsideHandler: function () {
+            if (this._clickOutsideHandler) {
+                this._clickOutsideHandler.detach();
+            }
+        },
+
+        /**
+         * Hides the row options
+         *
+         * @method _uiHideOptions
+         * @protected
+         */
+        _uiHideRowOptions: function () {
+            this.get('container').all(SELECTOR_ROW).removeClass(CLASS_ROW_SELECTED);
+
+            this._clearClickOutsideHandler();
+
+        },
+
+        /**
+         * Makes request for data
+         *
+         * @method _fireSearchEvent
+         * @protected
+         * @param event {Object} event facade
+         */
+        _fireSearchEvent: function () {
+            this.fire('contentSearch', {
+                viewName: 'all-content',
+                loadContentType: true,
+                search: {
+                    criteria: {SubtreeCriterion: this.get('rootLocation').get('pathString')},
+                    /*
+                     * @TODO sort items by modification date
+                     * see https://jira.ez.no/browse/EZP-24998
+                     *
+                     * sortClauses: {DateModifiedClause: 'DESC'},
+                     */
+                    limit: 10
+                },
+                callback: Y.bind(this._renderContentRows, this)
+            });
+        },
+
+        /**
+         * Renders content rows in a block
+         *
+         * @method _renderContentRows
+         * @protected
+         * @param error {Boolean} is error?
+         * @param list {Array} rows content data
+         */
+        _renderContentRows: function (error, list) {
+            var rows = list.map(function (data) {
+                return {
```

Comment:
@dpobel in case when there is only one variable introduced the indentation in the code above is correct. If there were more variables, then it would be incorrect.